toolchain/scripts/toolchain.sh

```diff
@@ -521,7 +521,11 @@ function generate_oasisctl() {
   version=$1
 
   log "[generate_oasisctl] Generate OasisCTL docs"
+  log "[generate_oasisctl] Invoking download_oasisctl"
```

Comment:
This should be moved inside the if

Reply:
Solved in https://github.com/arangodb/docs-hugo/pull/288/commits/841bec51be755533ad43ed2491e89bbb680d0e9d